### ISSUE TITLE
For Settings Screen, UIGrabInstancer on Individual Settings Cannot Be Instanced 

### ISSUE THREAD

**Issue by stiefeljackal** [Yellow-Dog-Man/Resonite-Issues#1706]
### Describe the bug?

The UIGrabInstancer on the individual settings (2x3 Vertical Dots) does not create an instance of the individual setting as a facet when trying to perform this action from the dash.

### To Reproduce

1. Open the Dash.
2. Go to Settings.
3. Locate any individual setting with the 2x3 Vertical Dots.
4. Attempt to grab the individual setting by hovering your laser (or cursor) over these dots and perform the grab interaction.

### Expected behavior

Should be able to instance an individual setting as a facet from the Dash.

### Screenshots

https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/20023996/21c65a46-6b41-4679-9499-217c8f4877eb

### Resonite Version Number

2024.4.15.1407

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Valve Index

### Log Files

[JACKAL_MK2 - 2024.4.15.1407 - 2024-04-15 23_15_16.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/14986467/JACKAL_MK2.-.2024.4.15.1407.-.2024-04-15.23_15_16.log)

### Additional Context

The Settings Screen example from `Resonite Essentials > Facets > Templates` demonstrates the correct behavior. You can perform the **To Reproduce** steps on this example to see the intended behavior. More than likely it is the nature of the Dash that is preventing an instance from being created.

### Reporters

* StiefelJackal
* JackTheFoxOtter

**Comment by JackTheFoxOtter** [comment]
I believe this is the same underlying issue as the one I reported here:
https://github.com/Yellow-Dog-Man/Resonite-Issues/discussions/1597#discussioncomment-9012433

**Comment by JackTheFoxOtter** [comment]
Taking my findings from that discussion here since that one is closed (and hard to parse since it's very long):

Okay, I think I figured it out. Grabbing values out of text fields works on all legacy screens, because those text fields aren't part of a facet. It looks like once the interaction is on a facet that's nested on a dash, grabbing it will grab and move the dash instead of picking out the value proxy. Putting a text field directly on the canvas of a dash page works as expected.

https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/35171546/a7879ee6-b8b4-4d8e-8349-ae112f7db1ed

This appeared to me as if all grab interactions on facets aren't properly passed down to the individual UI elements you want to interact with, resulting in multiple things not working (like the UI grab instancer or the default functionality of grabbing value proxies from text fields) when they are part of a facet.

**Comment by stiefeljackal** [comment]
> I believe this is the same underlying issue as the one I reported here: [#1597 (comment)](https://github.com/Yellow-Dog-Man/Resonite-Issues/discussions/1597#discussioncomment-9012433)

There it is. I will add you as a reporter.

**Comment by Frooxius** [comment]
I don't know, I'd need to investigate

**Comment by JackTheFoxOtter** [comment]
> Seeking feedback from @Frooxius - it appears that grabbing the dash is taking priority over the grab instancer- is this something that could be fixed on the template side by @RyuviTheViali by bumping the grab priority of the grab instancer/its grabbable component?

As stated, this also affects the ability to grab value proxies from text fields, which you can't manually specify a different grab priority for. This also affects user-made facets, so fixing this on a system level might be more desirable.

**Comment by Frooxius** [comment]
This is fixed in 2024.5.28.94, thanks!
